### ISSUE TITLE
Implementation Question

### ISSUE THREAD

**Issue by tedsecretsource** [GoogleChromeLabs/ProgressiveWordPress#6]
Hi,

Thanks for updating the readme. Everything works as expected now. I have a couple of follow up questions.

1. Is the twentyseventeen theme the PWA theme (it doesn't clarify which theme to activate in the instructions).
2. I set permalinks to post_name, clicked the Hello World post on the front end to make sure it loaded at least once, went back to the home page (on the front end), stopped the container (docker-compose stop), and then click the Hello World link (again). I was then expecting to see the page load (via all the fanciness of a PWA) but instead got ERR_CONNECTION_REFUSED. Is this the intended behavior?

Thanks again for working on this. As you can see, I'm closely following along.

**Comment by surma** [comment]
No, the theme’s name is “Surmblog”.

**Comment by tedsecretsource** [comment]
Thanks. Do I need to build the project to see the theme? I'm only seeing the default wordpress themes (following the `docker pull` instructions). Really sorry if this is obvious…

**Comment by surma** [comment]
No, it should show up right away. It does for me :-/

**Comment by tedsecretsource** [comment]
Hi again.

I've got it working now but over HTTP it returns 404 (which seems odd). HTTPS is fine.

I clicked the link to make the content available offline. I stopped docker and clicked the link to see Hello World. The theme/page loaded but instead of seeing the post content I saw: "This content is not available" Is that the expected behavior?

Thanks.

**Comment by surma** [comment]
Yes, you will only be able to read the post if you have opened it before while online or if you have clicked “Save for offline reading” while online.

During local development you can’t switch between HTTP and HTTPS. That’s a limitation of WordPress I have been running into.

**Comment by tedsecretsource** [comment]
> you will only be able to read the post if you have opened it before while online or if you have clicked “Save for offline reading” while online.

This is what I did but rather than seeing the actual content, I saw "This content is not available" message. I then tried the experiment again and ended up with a standard connection refused. Here's a video: https://youtu.be/0EwOQX9U6sM

I realize this is very much a WIP and that's fine. I'm just trying to help. If you'd rather I provide a different kind of help, just let me know :-)

Kind regards,

Ted Stresen-Reuter

**Comment by surma** [comment]
Looking at the video you have an HTTPS error. That means you didn’t add the [certificates](https://github.com/GoogleChromeLabs/ProgressiveWordPress/tree/master/caddy/certs) to your local keychain as instructed in the README. The certificate needs to be trusted by the system so that a ServiceWorker can be installed.

**Comment by tedsecretsource** [comment]
I've trusted it like this (see image below). Unless chrome uses it's own certificate chain/mechanism thing, I think I've got it configured properly.

![image](https://user-images.githubusercontent.com/28703641/31132553-f7294798-a854-11e7-99d3-6ac8e7be6b4f.png)

and this

![image](https://user-images.githubusercontent.com/28703641/31132739-7e4ef146-a855-11e7-9398-28f52610f751.png)

Which now, in the end, has given me this:

![image](https://user-images.githubusercontent.com/28703641/31132820-ae0c59dc-a855-11e7-8545-eed161a27c9c.png)

"Available for offline reading" - Bravo!
